gl-pvr.c

```diff
@@ -96,8 +96,8 @@ static inline void pvr_hdr_submit(const GLuint *src) {
 }
 #else
```

Comment:
Lets get rid of this #if 0 block while at it. The functions have been entirely superseded multiple times now. Additionally since the new changes mean that code shouldn't be directly using the SQs anymore it could lead people astray.